modules/mcuboot/Kconfig

```diff
@@ -39,10 +39,15 @@ config MCUBOOT_IMAGE_VERSION
 	default "0.0.0+0"
 	help
 	  Value to be passed as 'version' argument to 'imgtool.py' when
-	  creating signed image. Note that no semantics are connected to
-	  this variable. It does not provide downgrade prevention, and is only
-	  valuable for debugging purposes. Format: maj.min.rev+build with
-	  latter parts optional.
+	  creating signed image.
```

Comment:
```suggestion
	  creating the signed image.
```
maybe?

Reply:
creating a signed image.

I think this is best. Not a sentence I touched though, but you are right that was wierd.